ReactWindows/ReactNative/Bridge/JavaScriptBundleLoader.cs

```diff
@@ -1,179 +0,0 @@
-﻿using System;
-using System.Threading.Tasks;
-using Windows.Storage;
-using static System.FormattableString;
-
-namespace ReactNative.Bridge
-{
```

Comment:
It was originally a file move but GitHub shows it as a delete/add if you change any lines.